### PR TITLE
Set default font family "Monospace" and default font size for text and app bar title

### PR DESCRIPTION
**Pull request by amorenew**
This pull request for issue https://github.com/ammaratef45/zold-flutter-client/issues/94
I added a font file.
I set the default font family and default font size in main.dart theme block.
@ammaratef45 could you review it.?

### CODE REVIEW

**Comment by 0crat** [comment]
@ammaratef45[/z](https://www.0crat.com/u/ammaratef45) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#98](https://github.com/ammaratef45/zold-flutter-client/issues/98) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@ammaratef45[/z](https://www.0crat.com/u/ammaratef45) everybody who has role `REV` is banned at [#98](https://github.com/ammaratef45/zold-flutter-client/issues/98); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: zold_wallet/pubspec.yaml [hunk 1]
```diff
@@ -32,4 +32,10 @@ flutter:
...
   android: "launcher_icon" 
   ios: true
-  image_path: "assets/icon/icon.png"
\ No newline at end of file
+  image_path: "assets/icon/icon.png"
+
+  fonts:
+  - family: Monospace
+    fonts:
+      - asset: assets/fonts/Monospace.ttf
```
Comment (ammaratef45) [inline zold_wallet/pubspec.yaml]: @amorenew you should add the `Monospace.ttf` file to the assets, right?

File: zold_wallet/assets/fonts/Monospace.ttf [hunk 2]
```diff
@@ -0,0 +1 @@
+Free download fonts at http://sharefonts.net
```
Comment (ammaratef45) [inline zold_wallet/assets/fonts/Monospace.ttf]: @amorenew this is not the font file as I guess

**Review by ammaratef45 — COMMENTED** [review]
@amorenew please revise my comments

File: zold_wallet/pubspec.yaml (hunk 1 above)
Comment (amorenew) [reply zold_wallet/pubspec.yaml]: I fixed it and used Roboto Mono from google material
I found that Monospace is a general name for fonts style and matched monospaced font is Roboto Mono in material design fonts

File: zold_wallet/assets/fonts/Monospace.ttf (hunk 2 above)
Comment (amorenew) [reply zold_wallet/assets/fonts/Monospace.ttf]: @ammaratef45 Yes it was a mistake I fixed it and amended the commit. check the last commit in this pull request.

**Comment by ammaratef45** [comment]
@amorenew Thanks

**Comment by amorenew** [comment]
@ammaratef45 You are welcome.
Could you assign me 2 tasks this time?

**Comment by ammaratef45** [comment]
@amorenew sure
Check the opened issues and choose two tasks that you may implement,
I'll gladly assign them to you
FYI, you can suggest new things you think should be done too.

**Comment by 0crat** [comment]
Job `gh:ammaratef45/zold-flutter-client#98` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#98](https://github.com/ammaratef45/zold-flutter-client/issues/98) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Pull request merged**